enterprise/server/cmd/ci_runner/main.go

```diff
@@ -571,14 +581,47 @@ func bazelCommand() string {
 }
 
 func setupGitRepo(ctx context.Context) error {
+	repoDirInfo, err := os.Stat(repoDirName)
+	if err != nil && !os.IsNotExist(err) {
+		return status.WrapErrorf(err, "stat %q", repoDirName)
+	}
+	if repoDirInfo != nil {
```

Comment:
i think this could be if os.IsExist(err) && repoDirInfo != nil {

Reply:
The test actually fails with the `os.IsExist(err)` addition, so I reverted that bit.

Reply:
okay, sgtm!